tests/virt_autotest/virt_utils.pm

```diff
@@ -317,21 +318,18 @@ sub generate_guest_asset_name {
       . lc(get_required_var('SYSTEM_ROLE')) . '_'
       . get_required_var('ARCH');
 
-    record_info('Guest asset info', "Guest asset name is : $composed_name");
-
     return $composed_name;
 }
 
 sub get_guest_disk_name_from_guest_xml {
     my $guest = shift;
 
     # Our automation only supports single guest disk
-    my $disk_from_xml = script_output("virsh dumpxml $guest | sed -n \'/disk/,/\\\/disk/p\' | grep 'source file=' | grep -v iso");
+    my $disk_from_xml = script_output("virsh dumpxml $guest | sed -n \'/disk/,/\\\/disk/p\' | grep 'source file='");
```

Comment:
This can also be simpified as virsh dumpxml $guest | xmlstarlet  sel  -t -v //disk/source/@file. This is not only simple, but also tells what it is looking for, namely file attribute in source in disk

Reply:
tried, it worked. thanks.